src/os/bluestore/BlueStore.cc

```diff
@@ -6941,11 +6941,11 @@ void BlueStore::_txc_add_transaction(TransContext *txc, Transaction *t)
 
   _dump_transaction(t);
 
-  vector<CollectionRef> cvec(i.colls.size());
+  size_t coll_size = i.colls.size();
+  vector<CollectionRef> cvec(coll_size);
   unsigned j = 0;
-  for (vector<coll_t>::iterator p = i.colls.begin(); p != i.colls.end();
-       ++p, ++j) {
-    cvec[j] = _get_collection(*p);
+  for (; j < coll_size; j++) {
```

Comment:
er... I think only one variable changed with "for" loop, doesn't need p and j.

If you prefer current style, I am fine.